boilerplate/openshift/golang_osd_cluster_operator/standard.mk

```
@@ -71,8 +71,8 @@ docker-push: push
 
 .PHONY: gocheck
 gocheck: ## Lint code
-	boilerplate/openshift/golang_osd_cluster_operator/golint.sh
-	${GOENV} go vet ./cmd/... ./pkg/...
+	boilerplate/openshift/golang_osd_cluster_operator/ensure.sh golangci-lint
+	GOLANGCI_LINT_CACHE=/tmp/golanci-cache golangci-lint run -c boilerplate/openshift/golang_osd_cluster_operator/golangci.yml ./...
```

nit:
```suggestion
	GOLANGCI_LINT_CACHE=/tmp/golangci-cache golangci-lint run -c boilerplate/openshift/golang_osd_cluster_operator/golangci.yml ./...
```
I'm curious how this env var is used by golangci-lint. If you don't set it, does it use a default? (I couldn't find documentation for it at a quick glance.)

---

I'm not sure why it resolves to `/.cache` as the default directory for caching but this [issue](https://github.com/golangci/golangci-lint/issues/734) summarizes what's going on.